test_bot/test_bot.py

```diff
@@ -38,7 +38,8 @@ def download_sf():
 
 
 def download_lc0():
-    response = requests.get("https://github.com/LeelaChessZero/lc0/releases/download/v0.28.2/lc0-v0.28.2-windows-cpu-dnnl.zip", allow_redirects=True)
+    response = requests.get("https://github.com/LeelaChessZero/lc0/releases/download/v0.28.2/lc0-v0.28.2-windows-cpu-dnnl.zip",
+                            allow_redirects=True)

```

Comment:
You can split the string in the first argument the same way as in the lichess-bot edits to get under the line length limit.

Reply:
There is no reason to do this, since it is already under the line limit.

The flake8 limit is 127 characters. Maybe your IDE has a different limit (mine is 120), because this change is enough to fix the flake8 error.